vector/src/test/java/im/vector/app/screenshot/PaparazziExampleScreenshotTest.kt

```diff
@@ -31,6 +31,7 @@ class PaparazziExampleScreenshotTest {
     val paparazzi = Paparazzi(
             deviceConfig = PIXEL_3,
             theme = "Theme.Vector.Light",
+            maxPercentDifference = 0.0,
```

Comment:
Ah, I thought it will be a global setting :/

Reply:
we could create our own function to create the Paparazzi instance (to act as a global)

Reply:
Yes, I have done it on a stash :), using a parent class:

<img width="368" alt="image" src="https://user-images.githubusercontent.com/3940906/192497538-afae23ee-39dc-4c57-8b9c-5146081406e5.png">

Also maybe update [the doc](https://github.com/vector-im/element-android/blob/7740404b1430d32282ed7289db2c1bba8e107af7/docs/screenshot_testing.md#example) with the new param.